modules/apps/roles/roles-selector-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -166,11 +166,14 @@ String methodName = null;
 </liferay-util:buffer>
 
 <%
+PortletURL clearResultsURL = (PortletURL)request.getAttribute("edit_roles.jsp-portletURL");
+clearResultsURL.setParameter("keywords", StringPool.BLANK);
```

Comment:
Brian usually prefers a line break between a variable declaration and its usage.

@hhuijser the SF tests passed on the previous pull, is this expected? https://github.com/SamZiemer/liferay-portal/pull/771#issuecomment-722076228